python/graphstorm/trainer/gsgnn_trainer.py

```diff
@@ -31,6 +31,12 @@
 class GSgnnTrainer():
     """ Generic GSgnn trainer.
 
+    This class is used as a mixin for classes that implement trainers
+    for various learning tasks at the node and edge level.
+
+    It contains functions that can be used in the implementing classes'
+    `fit` and `eval` functions.
```

Comment:
Could you add a the end, "If users would like to implement their own trainers, please extend this class and implement both `fit` and `eval` functions."?